services/data_plane_manager/src/main/java/com/futurewei/alcor/dataplane/client/ZetaGatewayClient.java

```diff
@@ -0,0 +1,229 @@
+package com.futurewei.alcor.dataplane.client;
+
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.futurewei.alcor.common.db.CacheException;
+import com.futurewei.alcor.common.entity.ResponseId;
+import com.futurewei.alcor.common.enumClass.StatusEnum;
+import com.futurewei.alcor.common.executor.AsyncExecutor;
+import com.futurewei.alcor.dataplane.cache.VpcGatewayInfoCache;
+import com.futurewei.alcor.dataplane.client.grpc.DataPlaneClientImpl;
+import com.futurewei.alcor.dataplane.entity.MulticastGoalState;
+import com.futurewei.alcor.dataplane.entity.UnicastGoalState;
+import com.futurewei.alcor.dataplane.entity.ZetaPortGoalState;
+import com.futurewei.alcor.schema.Common;
+import com.futurewei.alcor.web.entity.dataplane.InternalPortEntity;
+import com.futurewei.alcor.web.entity.gateway.*;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.http.HttpEntity;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.MediaType;
+import org.springframework.stereotype.Service;
+import org.springframework.web.client.RestTemplate;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.concurrent.*;
+
+@Service("zetaGatewayClient")
+public class ZetaGatewayClient {
+    private static final Logger LOG = LoggerFactory.getLogger(DataPlaneClientImpl.class);
+
+    @Autowired
+    private VpcGatewayInfoCache gatewayInfoCache;
+
+    @Value("${zetaGateway.check.timeout}")
+    private String zetaGatewayCheckTimeout;
+
+    @Value("${zetaGateway.check.interval}")
+    private String zetaGatewayCheckInterval;
+
+    @Value("${microservices.zeta.service.url}")
+    private String zetaGatewayUrl;
+
+    @Value("${microservices.gateway.service.url}")
+    private String gatewayUrl;
```

Comment:
What is the difference between zeta.service.url and gateway.service.url?

Reply:
zeta.service.url is the url for zeta gateway management plane
gateway.service.url is the url for gateway manager

Reply:
Got it. I would suggest to rename microservices.zeta.service.url to microservices.zeta.management.url.

Reply:
ok, microservices.zeta.service.url has renamed to microservices.zeta.management.url.